lib/matplotlib/backend_bases.py

```diff
@@ -3162,7 +3162,7 @@ def trigger_tool(self, name):
 
         Parameters
         ----------
-        name : string
+        name : str
```

Comment:
also add missing final dots in the sentences below? (throughout the docstrings you edited here)

Reply:
Trying to not get carried away too far. Otherwise the PR gets endless and even more terrible to review. These sentences need rephrasing anyway. It's better to leave them alone for now and do everything in one go when I come back.